app/app/dialog/[id]/backup/[cid]/page.tsx

```diff
@@ -319,7 +319,7 @@ export default function Page() {
   }
 
   return (
-    <Layouts isSinglePage withHeader={false}>
+    <Layouts isSinglePage withHeader={restoredBackup.loading}>
```

Comment:
it’s nice that it gives users a way to go back to the previous page, but I think it would feel more polished if we showed the actual chat header right away. We already have the dialog info (image, type, and name), so we could use that to avoid showing an empty header at first. 
Right now, it looks kinda glitch when the real header suddenly loads in.

Reply:
yes, i noticed that glitch too. so, i'll render the chat header as is

Reply:
okay, @BravoNatalie, i've updated this. 

although, there's this nanosecond flash of a blank page that is due to this: 

```ts
 useEffect(() => {
    const fetchBackup = async () => {
      const userId = await getMe()
      if (!userId) return
      setUserId(userId)
      await restoreBackup(backupCid!, normalizedId, 20)
    }

    fetchBackup()
  }, [backupCid, restoreBackup, getMe])
  ```
  
  before the component renders

Reply:
I'm getting the header, but it shows the previous header loaded first
https://github.com/user-attachments/assets/e8c2a4e8-28d4-47b2-af8c-2a2fc190d9e3

Reply:
ah, i see it now.